Overview.src.html

```diff
@@ -934,6 +934,24 @@ <h4 id=terminology-headers>Headers</h4>
 Unless stated otherwise, it is unset.
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-use-token-binding>use-token-binding flag</dfn>.
+Unless stated otherwise, it is unset.
+<p class="note no-backref"> This flag indicates whether the User Agent will send its Token Binding
+Id for the origin along with the request when it transmits the request to the server.
```

Comment:
No, the use-token-binding flag is set for all requests per step 14 in HTTP-network-or-cache fetch.

A server S sending a redirect-response (aimed at server R) can ask the UA to send the Token Binding Id for S to R only if the UA used token binding with S (indicated by this flag)

Then, if the UA uses token binding with R, the UA will send the Token Binding Id for S along with the Token Binding Id for R.